packages/unit-vitest/package.json

```
@@ -39,18 +39,18 @@
   },
   "devDependencies": {
     "@types/lodash-es": "^4.17.12",
-    "@vitest/ui": "^2.0.5",
-    "@vue/test-utils": "^2.4.4",
+    "@vitest/ui": "^3.0.8",
+    "@vue/test-utils": "^2.0.5",
     "quasar": "^2.14.6",
     "rimraf": "^5.0.5",
-    "vitest": "^2.0.5",
+    "vitest": "^2.4.6",
     "vue": "^3.4.21"
   },
   "peerDependencies": {
     "@vitest/ui": "^1.1.0 || ^2.0.0",
     "@vue/test-utils": "^2.4.3",
     "quasar": "^2.12.7",
-    "vitest": "^1.1.0 || ^2.0.0",
+    "vitest": "^3.0.8",
```

This will only allow Vitest v3. I previously said to add v3 to the range, not use it as the only thing.
```suggestion
    "vitest": "^1.1.0 || ^2.0.0 || ^3.0.0",
```